docker/release/fedora27/Dockerfile

```diff
@@ -22,6 +22,9 @@ LABEL maintainer="PowerShell Team <powershellteam@hotmail.com>" \
 
 # TODO: addd LABEL org.label-schema.vcs-ref=${VCS_REF}
 
+# Apply any security updates to the image
+RUN dnf upgrade-minimal -y --security
```

Comment:
you should run `dnf clean all` before you write the layer (the end of the run statement)

Reply:
Updated.